reportsServer.js

```diff
@@ -23,6 +28,16 @@ function getPageSize(pageSize) {
   }
 }
 
+function getPageSizeByOrientation(pageSize, orientation) {
+  const size = getPageSize(pageSize);
+  if (orientation && orientation === PAGE_ORIENTATION.landscape) {
+    const h = size.height;
+    const w = size.width;
+    return { width: h, height: w };
```

Comment:
shouldn't it be the opposite? you assigned h to width and w to height..

Reply:
and no need for the h, w consts..

Reply:
if the orientation is landscape then the width and height are switched.
![image](https://user-images.githubusercontent.com/18641362/35090488-78d9cf42-fc42-11e7-8376-a951f71000d6.png)
cant without the consts..

Reply:
👍